### ISSUE TITLE
"Manage localhost" doesn't show after navigating to other nets

### ISSUE THREAD

**Issue by nathanleclaire** [workbenchapp/solana-workbench#255]
re: https://github.com/workbenchapp/solana-workbench/commit/8f7ab0b14c95911ccad29fc8645852ae45dcf855

I noticed that if you click another net, that menu item changes back to "localhost" 

Don't feel this is particularly high pri for 0.4.0, but wanted to note it down

**Comment by SvenDowideit** [comment]
yeah - its because we don't have a validator status for localnet if we're not on localnet - i'm only changing the link to the manage page _if_ we know the validator's down, 

i think i'd like to make the dropdown more complicated, instead of doign this in the menu, but v0.5.0 :)
